test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1001,7 +1015,7 @@ def test_cgroup_qstat_resources(self):
         cput1 = qstat1[0]['resources_used.cput']
         mem1 = qstat1[0]['resources_used.mem']
         vmem1 = qstat1[0]['resources_used.vmem']
-        time.sleep(10)
+        time.sleep(15)
```

Comment:
Be kind to the person running the test.  Add a log message to explain why you are sleeping.

Reply:
Done.